scripts/corr2_dsim_control.py

```diff
@@ -131,7 +133,7 @@
         sinesource.set(scale=xscale, frequency=yfreq)
         print "Sine Source:", sinesource.name
         print "Scale:", sinesource.scale
-        print "Frequency:", sinesource.frequency        
+        print "Frequency:", sinesource.frequency
     except AttributeError:
         print "You can only select between, sin_0 or sin_1"
```

Comment:
You should get the names from the dhost.sine_sources object, i.e. say 

    print "You can only select between sine sources: {}".format(sorted(
        ss.name for ss in dhost.sine_sources))